### ISSUE TITLE
How to use conditions? 

### ISSUE THREAD

**Issue by illogicalextend** [joshuamsmith/ConnectPyse#13]
Does this library support using conditions? 

eg. if I call to this endpoint, I can append conditions to the end so I only get back relevant data:
v4_6_release/apis/3.0/service/tickets?conditions=priority/id=7

the get_tickets() within tickets_api.py doesn't seem to accept any further parameters, right?

**Comment by joshuamsmith** [comment]
Yes it does support conditions. Object instances have properties that can be set ('conditions', 'orderBy', 'childconditions', 'customfieldconditions', 'page', 'pageSize') and will be applied during API calls.
You can get more information about each parameter here: https://developer.connectwise.com/Products/Manage/Developer_Guide#Query_String_Parameters.

So for your example you'd create an instance of Ticket and assign it a value for the parameters you want:
ticket.conditions = "priority/id=7" and then run ticket.get_tickets()

or something to that effect as it's been a bit.

**Comment by billcd** [comment]
In addition, if you need to search for multiple conditions, they can be separated by AND and OR logic operators.  For instance, if you wanted to get all Open tickets for the resource jane the condition should be: `ticket.conditions =  'resources contains "jane" AND closedFlag = False'`
